### DISCUSSION TITLE
StandardListViewItem functionalities

### DISCUSSION THREAD

**Discussion by csavels** [slint-ui/slint#3868] [Ideas]
Hello,

Would it be possible to provide these functionalities for "StandardListViewItem"?

- a callback like "`current-row-changed(row)"` but for the column: "`current-column-changed(col)"` OR /AND a "`current-selection-changed(row, col)"` for example.

	- add a "**color**" and "**font-size**" field for the "`StandardListViewItem`" text. 

Thank you very much for your work, which is helping me to learn Rust.

**Comment by ghost** [comment]
Hello,

do you mean something like a cell selection on the `StandardTableView`?

The styling customization of widgets is still ongoing you can check the following issue: https://github.com/slint-ui/slint/issues/45
